exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,35 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+if (num < 0){
+  return true
+}else {
+  return false;
+}
+}
```

Comment:
This is good and is correct! But there is a way to write simple if/else statements in one line if you are just returning true/false:
```
function isNegative(num) {
  return num < 0;
}
```
This works because the function returns `true` if the `num < 0` or `false` if not

Reply:
Thank you Gladwin. I've been practising with this a lot. I think I'm beginning to get the hang of it. I was working with some classmates and we practiced this a lot. I feel more comfortable when I work the long way, get the test working,  then try to do it the shorter way. It's long winded but helps me to 'see' what i'm doing.